queries.sql

```diff
@@ -8,3 +8,26 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > '10.5';
 SELECT * FROM animals WHERE neutered = 't';
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN '10.4' AND '17.3';
+BEGIN;
+UPDATE animals SET species = 'unespecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
```

Comment:
- [ ] Add SQL code here to verify that changes were made. It can look something like this:
```sql
SELECT species from animals
```
- [ ] Please add this transaction as mentioned in the requirements where you are to delete all records and rollback. It can look something similar to this:
 ```sql
BEGIN;

DELETE FROM animals WHERE ;
SELECT COUNT(*) FROM ANIMALS;

ROLLBACK;
SELECT COUNT(*) FROM ANIMALS;
```